proto/finalitygadget.proto

```diff
@@ -10,6 +10,11 @@ service FinalityGadget {
   rpc QueryIsBlockBabylonFinalized(QueryIsBlockBabylonFinalizedRequest)
       returns (QueryIsBlockFinalizedResponse);
 
+  // QueryIsBlockBabylonFinalizedFromBabylon returns the finality status of a block by
+  // querying Babylon chain directly
+  rpc QueryIsBlockBabylonFinalizedFromBabylon(QueryIsBlockBabylonFinalizedRequest)
+      returns (QueryIsBlockFinalizedResponse);
```

Comment:
this seems irrelevant to the PR

do you know why we didn't include it? IIRC we intentionally removed this to prevent DoS attack. plus, there is no call site of this grpc method anyway

Reply:
we can remove it from this PR and add it to a new one. It is needed for debugging the isBlockFinalized call from Babylon. I was trying to call it during most recent debug of devnet-7 but realized we never exposed the grpc method.

Reply:
> we can remove it from this PR and add it to a new one

it's fine to keep it here but the question is whether we should expose it. probably fine for now but we need to be aware of the potential DoS attacks

does calling this function making babylon rpc calls every time or it's reading from the db? if it's calling the Babylon rpc, then there is a DoS attack vector